backend/src/main/java/eu/clarin/switchboard/core/MediaLibrary.java

```diff
@@ -66,8 +48,8 @@
         Path path; // actual path on disk
         long fileLength;
 
-        String mediatype;
-        String language;
+        Profile profile;
+        List<Profile> secondaryProfiles;
```

Comment:
Shouldn't be too hard to refactor these to have both getters and setters, and then the fields can be private.

Reply:
I now see that the parent class suffers from the same thing